### ISSUE TITLE
TODAY marker in Calendar is VERY difficult to see in Dark Mode

### ISSUE THREAD

**Issue by muchqs** [tutao/tutanota#6273]
- [ ] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Bug in web app**
**Bug in mobile app**

**Describe the bug**
The circle marker marking TODAY's date in Calendar is VERY difficult to see in Dark Mode. On desktop and mobile.
It used to have a green accent like the Tuta logo on the top-left, but now it's grey and blends into the background making it very difficult to see.
Please see this pic. Can you spot what day is it instantly??
https://i.imgur.com/c1aoF23.png

more pix:
[old version where the Today's date is circled in a light green color.](https://i.imgur.com/pHLrBuQ.png)
[new version where Today's date is difficult to see.](https://i.imgur.com/S1k5cmu.jpg)

**To Reproduce**
open calendar in dark mode.

**Expected behavior**
Please return the previous color accent.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by charlag** [comment]
We need to address this, yes

**Comment by charlag** [comment]
Covered by #6562
